### PR TITLE
fix: remove disabled dials from output

### PR DESCRIPTION
**Pull request by dimsome**
added check to remove looping over disabled dials. Disabled dials will not produce a console output anymore.

### CODE REVIEW

File: tasks/dials.ts [hunk 1]
```diff
@@ -115,9 +115,9 @@ const dialsDetailsToString = (dialsDetails: Array<DialDetails>) =>
     dialsDetails
         .map(
             (dd, i) =>
-                `${dialNames[i].padStart(21)}\t${usdFormatter(dd.voteWeight, 18, 5, 2)}\t${usdFormatter(dd.distributed)}\t${usdFormatter(
-                    dd.donated,
-                )}\t ${usdFormatter(dd.rewards)}`,
```
Comment (doncesarts) [inline tasks/dials.ts:120]: you can remove the index "i" , it is not used.
but question how it avoids the disabled ones ? i would expect a filter or similar

Comment (dimsome) [reply tasks/dials.ts:120]: right, forgot. The `i` was used before.

The filter happens in the calculation, disabled dials are skipped in the loop. Line 225

File: tasks/dials.ts [hunk 2]
```diff
@@ -221,10 +221,11 @@ task("dials-snap", "Snaps Emissions Controller's dials")
```
Comment (doncesarts) [inline tasks/dials.ts:221]: it is just styling but i would rather see a filter latestDialVotes.filter().forEach
than a return inside the foreach

Comment (dimsome) [reply tasks/dials.ts:223]: Not really possible on that array without creating mismatched indexes, but reworked to add a filter where the string is built.

**Pull request merged**